chef.gemspec

```diff
@@ -15,13 +15,13 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = ">= 2.0.0"
 
-  s.add_dependency "chef-config", "= #{Chef::VERSION}"
+  s.add_dependency "chef-config", "= 12.4.1"
```

Comment:
can't you just release chef-config 12.4.2 at the same time and avoid additional confusion here?